app/models/manageiq/providers/ansible_tower/shared/automation_manager/azure_credential.rb

```diff
@@ -0,0 +1,17 @@
+module ManageIQ::Providers::AnsibleTower::Shared::AutomationManager::AzureCredential
+  extend ActiveSupport::Concern
+
+  COMMON_ATTRIBUTES = {
+  }.freeze
+
+  EXTRA_ATTRIBUTES = {
+  }.freeze
+
+  API_ATTRIBUTES = COMMON_ATTRIBUTES.merge(EXTRA_ATTRIBUTES).freeze
```

Comment:
These 3 default empty hashes are already inherited from `Credential` as in [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/automation_manager/credential.rb#L2) and [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/embedded_ansible/automation_manager/credential.rb#L3) through [this](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/ansible_tower/shared/automation_manager/credential.rb#L75).

Similarly all the others in this PR don't need them.